cmp/clock/Clock.js

```diff
@@ -0,0 +1,135 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2019 Extremely Heavy Industries Inc.
+ */
+
+import {box, span} from '@xh/hoist/cmp/layout';
+import {hoistCmp, HoistModel, managed, useLocalModel, XH} from '@xh/hoist/core';
+import {action, observable} from '@xh/hoist/mobx';
+import {MINUTES, ONE_SECOND} from '@xh/hoist/utils/datetime';
+import {Timer} from '@xh/hoist/utils/async';
+import {isEmpty, isNil, isNumber} from 'lodash';
+import moment from 'moment';
+import PT from 'prop-types';
+
+/**
+ * A component to display the current time.
```

Comment:
Would note support for a configurable timezone here in the header comment - it's a special enough feature I think it's worth calling out. We could also note that offsets are loaded from the server, so we don't need to bundle a big TZ database with the client JS code.